contracts/VTokenInterfaces.sol

```diff
@@ -259,6 +265,11 @@ abstract contract VTokenInterface is VTokenStorage {
      */
     event SwapBorrowRateMode(address account, uint256 swappedBorrowMode);
 
+    /**
+     * @notice Event emitted when a borrow rate mode is swapped for account with amount
+     */
+    event SwapBorrowRateModeWithAmount(address account, uint256 swappedBorrowMode, uint256 amount);
```

Comment:
I think we could use the same `SwapBorrowRateMode` event, adding the swapped amount and the left amount to that event. That way we would simplify the analysis of these events

Reply:
Refactored, as well as for this [comment](https://github.com/VenusProtocol/isolated-pools/pull/157#discussion_r1112211425). Can verify the changes at https://github.com/VenusProtocol/isolated-pools/pull/157#discussion_r1112211425